core/utils/xml.ts

```diff
@@ -112,6 +112,24 @@ export function createTextNode(text: string): Text {
   return document.createTextNode(text);
 }
 
+/**
+ * Converts an XML string into a DOM structure.
+ *
+ * @param text An XML string.
+ * @returns A DOM object representing the singular child of the document
+ *     element.
+ * @throws if the text doesn't parse.
+ * @alias Blockly.utils.xml.textToDom
```

Comment:
You don't need `alias` anymore (and I'm working on deleting it everywhere else)

Reply:
Done.